packages/env/lib/download-sources.js

```diff
@@ -123,16 +123,11 @@ async function downloadGitSource( source, { onProgress, spinner, debug } ) {
 		await git.clone( source.url, source.clonePath, {
 			'--depth': '1',
 			'--no-single-branch': null,
+			'--branch': source.ref,
 		} );
 		await git.cwd( source.clonePath );
 	}
 
-	log( 'Fetching the specified ref.' );
-	await git.fetch( 'origin', source.ref );
```

Comment:
Unless I'm missing something, I think we still need these. :thinking: Otherwise, if the repo already exists (e.g. `isRepo`) wouldn't it not get updated if the tag was changed? (Example: you have gutenberg wp-env working locally and we update the version of tt1 which gets cloned. Your setup would never run the fetch / checkout for that new version)

Reply:
> Example: you have gutenberg wp-env working locally and we update the version of tt1 which gets cloned. Your setup would never run the fetch / checkout for that new version

That was the problem people were experiencing in the first place though, no?

But yeah, I will say that I tried repro'ing the problem locally on GB `trunk` (by switching the `theme-experiments` repo in `~/.wp-env` to the `tt1-blocks@0.4.3` tag, and deleting the `tt1-blocks@0.4.5` one) but couldn't really...

Reply:
Yeah, i was never able to replicate the problem either :(